common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4645,6 +4645,11 @@
         "message" : [
           "TRANSFORM with SERDE is only supported in hive mode."
         ]
+      },
+      "LAMBDA_FUNCTION_WITH_PYTHON_UDF" : {
+        "message" : [
+          "Python UDFs with a lambda function at a higher order function is unsupported. However, a Python UDF <funcName> was found."
```

Comment:
The full error message starts with `The feature is not supported: `, so how about `Python UDF <name> within a lambda function.`

Reply:
or `Lambda function with Python UDF <name>`, to be more consistent with the error condition name.